### PR TITLE
Refactor: use explicit setup functions instead of importing from app

### PR DESCRIPTION
**Pull request by simonihmig**

### CODE REVIEW

File: addons/core/addon-test-support/setup.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export function setupDictionary(dictionary: ConvertersOpinionatedRecord): void {
+  // @todo
+}
+
+export function setupLabelOverrides(labelOverrides: LabelOverridesRecord): void {
+  // @todo
+}
+
+export function setupSteps(steps: StepImplementationsRecord): void {
```
Comment (lolmaus) [inline addons/core/addon-test-support/setup.ts]: Hmm, I would rather put these in corresponding files in `core`: `dictionary.ts`, `annotations.ts`, etc.

They all could share the same tooling for store.

**Review by lolmaus — COMMENTED** [review]
Nice!

File: addons/core/addon-test-support/setup.ts (hunk 1 above)
Comment (simonihmig) [reply addons/core/addon-test-support/setup.ts]: Ok, I can do that. But at the end this is a private implementation detail. You shouldn't import from `/setup` anyway. 

The important piece is the public API we are now exposing. Are you d'accord with that?

**Review by lolmaus — APPROVED** [review]
Nice!!

**Pull request merged**